openstack-example-tests.spec

```diff
@@ -0,0 +1,79 @@
+%{!?upstream_version: %global upstream_version %{version}%{?milestone}}
+
+%global pname example_tests
+
+%global service example-tests
+Name:           openstack-%{service}
+Version:        XXX
+Release:        XXX
+Summary:        Example Test Framework
+License:        ASL 2.0
+URL:            http://launchpad.net/%{service}/
+
+Source0:        http://tarballs.openstack.org/%{service}/%{service}-master.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  python2-devel
+BuildRequires:  python-pbr
+BuildRequires:  python-setuptools
+BuildRequires:  git
+
+Requires:       python-pbr
+Requires:       python-setuptools
+
+# test dependencies requirements
+BuildRequires:  python-hacking
+BuildRequires:  python-mock
+BuildRequires:  python-coverage
+
+%description
+This project contains example test framework.
+
+%package -n python-%{service}-doc
```

Comment:
I'm not sure this is the best subpackage name for an openstack-service-tests package. Shouldn't it be openstack-%{service}-test-doc or python-%{service}-test-doc?